src/app.tsx

```diff
@@ -98,6 +99,7 @@ export const Application = () => {
 
         // On initial load redirect to the users home directory
         cockpit.user().then(user => {
+            setUser(user);
```

Comment:
Note from @martinpitt don't render when user is not intialised

Reply:
So we could change the below condition to `if (loading && user)` instead of adding `user` to `setEffect`